src/GameLoop.tsx

```diff
@@ -105,7 +104,7 @@ export function GameLoop() {
     const [_keydownEvent, _setKeydownEvent] = useState(() => {
         globalThis.addEventListener(
             "keydown",
-            updatePlayerPos.bind(this, playerPhysics, boardPhysics),
+            updatePlayerPos.bind(globalThis, playerPhysics, boardPhysics),
```

Comment:
What's this needed for?

Reply:
Good catch, it's not needed anymore. Removed the `bind`

Reply:
Also you can just use a closure instead of `bind`.  I think it type checks better and/or is more performant.

Reply:
Ah I see. Thanks!